src/components/Main/Form/Form.component.jsx

```diff
@@ -16,40 +16,44 @@ const Form = props => {
     }));
   };
 
-  const handleSubmit = e => {
-    e.preventDefault();
-  };
-
-  const inputs = props.inputs
-    ? props.inputs.map(input => {
-        const inputItem = (
-          <Input
-            {...input}
-            key={input.id}
-            value={input.state ? input.state[input.id] : state[input.id]}
-            onChange={handleChange}
-            className={input.className}
-          />
-        );
-        if (props.header === 'Параметры опроса') {
-          return (
-            <tr key={input.id ? input.id : null} className="paramsInputs">
-              <td className="paramsInputs-item-wrapper">
-                {inputItem}
-                <label htmlFor={input.id}>{input.name}</label>
-              </td>
-            </tr>
-          );
+  const inputs = props.inputs?.map(input => {
+    const inputItem = (
+      <Input
+        {...input}
+        key={input.id}
+        value={
+          input.state
```

Comment:
Some notions to keep in mind: 
1. Single Source of Truth
2. Controlled vs Uncontrolled form inputs
https://reactjs.org/docs/forms.html
https://reactjs.org/docs/uncontrolled-components.html

Reply:
refactored that, now use only state from props for this component